unfoldingWord/translationCore
# Multiple Fixes Stepper Icon Colors

#### This pull request addresses:
This PR address issues with the stepper icon colors not persisting after going through the workflow,
Fix for the initial stepper icon color
& removal of legacy code

#### How to test this pull request:
Go through the stepper and ensure the following is true for both the home stepper and project import:
The current step is black
Steps that are not available are grey
Steps that are available are blue (besides the current one)


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3285)
